src/api/worker_api.rs

```diff
@@ -151,6 +152,67 @@ impl WorkerApiClient {
             Err(response.error_for_status().unwrap_err())
         }
     }
+
+    /// Sends the completed computation file to the worker host.
+    ///
+    /// This method transmits the computed file containing task results, signatures,
+    /// and metadata to the worker API. The computed file is sent as JSON in the
+    /// request body, allowing the worker to verify and process the computation results.
+    ///
+    /// # Arguments
+    ///
+    /// * `authorization` - The authorization token/challenge for authenticating the request
```

Comment:
authorization is different from authentication.
What do we think of:
```suggestion
    /// * `authorization` - The authorization token/challenge to validate the request on the worker side
```

Reply:
Resolved in [f014b3a](https://github.com/iExecBlockchainComputing/tee-worker-post-compute-rust/pull/13/commits/f014b3a66de47ed60956ce0d044a1db30ae215bc)